src/transformers/models/llava_next/modeling_llava_next.py

```diff
@@ -648,7 +648,7 @@ def _merge_input_ids_with_image_features(
 
         return final_embedding, final_attention_mask, position_ids, final_labels, final_input_ids
 
-    def pack_image_features(self, image_features, image_sizes, image_newline=None):
+    def pack_image_features(self, image_features, image_sizes, vision_feature_select_strategy, image_newline=None):
```

Comment:
Should it have `"default"` as a default kwarg value so as to not change the signature and keep usages of `pack_image_features` outside of the model working?

(Not necessary if this method should be considered internal)

Reply:
yes, should be internal as it doesn't make much sense outside of LLaVA